pootle/apps/pootle_fs/management/commands/fs_commands/rm.py

```diff
@@ -19,4 +19,4 @@ def add_arguments(self, parser):
             "--force",
             action="store_true",
             dest="force",
-            help=("Stage files and stores that are conflicting for removal"))
+            help=("Stage files and/or stores for removal"))
```

Comment:
this needs to retain the "that are conflicting" and in fact probs needs to have the following added "that are conflicting or untracked"